consumers/email-consumer/src/email_consumer/email_consumer.py

```
@@ -0,0 +1,46 @@
+import os
+
+import requests
+
+from base_consumer import BaseKafkaConsumer
+from lib.generate_html_verification_email import generate
+
+
+class EmailConsumer(BaseKafkaConsumer):
+    """Kafka consumer for handling email messages."""
+
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)